---
title: "FlowSorted.BloodExtended.EPIC"
author: "Lucas A Salas"
date: "`r Sys.Date()`"
output: rmarkdown::html_vignette
vignette: >
    %\VignetteIndexEntry{FlowSorted.BloodExtended.EPIC}
    %\VignetteEngine{knitr::rmarkdown}
    %\VignetteEncoding{UTF-8}
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(cache = FALSE, warning = FALSE, message = FALSE, 
                        cache.lazy = FALSE,collapse = TRUE, comment = "#>"
)
```

The FlowSorted.BloodExtended.EPIC package contains data derived from Illumina 
HumanMethylationEPIC DNA methylation microarrays (Salas LA et al. 2021), 
consisting of 56 blood cell references and 12 mixed blood samples, formatted
as an RGChannelSet object for integration and normalization using most of the 
existing Bioconductor packages. 

The FlowSorted.BloodExtended.EPIC dataset includes information from neutrophils 
(Neu, n=6), eosinophils (Eos, n=4), basophils (Bas, n=6), monocytes (Mono, n=5),
B naïve cells (Bnv, n=4), B memory cells (Bmem, n=6), T-helper CD4+ naïve cells 
(CD4nv, n=5), T-helper CD4+ memory cells (CD4mem, n=4), T regulatory cells 
(Treg, n=3), T-cytotoxic CD8+ naïve cells (CD8nv, n=5),T-cytotoxic memory CD8+ 
cells (CD8mem, n=4), and natural killer cells (NK, n=4), plus 12 DNA 
artificial mixtures, labeled as MIX in this dataset.  

Researchers may find this package useful as these samples represent different 
cellular populations, including eosinophils, basophils, monocytes, B naïve 
cells, B memory cells, T-helper CD4+ naïve cells, T-helper CD4+ memory cells, T 
regulatory cells, T-cytotoxic CD8+ naïve cells, T-cytotoxic memory CD8+ cells 
and natural killer cells, of cell sorted blood generated with high purity 
estimates. 
As a test of accuracy 12 experimental mixtures were reconstructed using fixed 
amounts of DNA from purified cells. 

**Objects included:**  
 1. *FlowSorted.BloodExtended.EPIC* is the RGChannelSet object containing the 
 reference library
 
```{r eval=FALSE}
library(ExperimentHub)  
hub <- ExperimentHub()  
query(hub, "FlowSorted.BloodExtended.EPIC")  
FlowSorted.BloodExtended.EPIC <- hub[["*****"]]  
FlowSorted.BloodExtended.EPIC  
```
 The raw dataset is hosted in both ExperimentHub (******) and GEO 
 [GSE167998](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167998)  

 2. *IDOLOptimizedCpGsBloodExtended* the IDOL L-DMR library for EPIC arrays  
```{r eval=FALSE}
head(IDOLOptimizedCpGsExtended)  
```  
 3. *IDOLOptimizedCpGsBloodExtended450k* the IDOL L-DMR library for legacy 450k 
 arrays  
```{r eval=FALSE}
head(IDOLOptimizedCpGsBloodExtended450k)  
```  
 *See the object help files for additional information*  
**References**

LA Salas et al. (2021) Enhanced cell deconvolution of peripheral blood using 
DNA methylation for high-resolution immune profiling. (Under review)

LA Salas et al. (2018). An optimized library for reference-based deconvolution 
of whole-blood biospecimens assayed using the Illumina HumanMethylationEPIC
BeadArray. Genome Biology 19, 64. doi:
[10.1186/s13059-018-1448-7](https://dx.doi.org/10.1186/s13059-018-1448-7).  

DC Koestler et al. (2016). Improving cell mixture deconvolution by
identifying optimal DNA methylation libraries (IDOL). BMC bioinformatics.
17, 120. doi:
[10.1186/s12859-016-0943-7](https://dx.doi.org/10.1186/s12859-016-0943-7).  

EA Houseman et al. (2012) DNA methylation arrays as surrogate
measures of cell mixture distribution. BMC Bioinformatics 13, 86.
doi: [10.1186/1471-2105-13-86](https://dx.doi.org/10.1186/1471-2105-13-86).  

[minfi](http://bioconductor.org/packages/release/bioc/html/minfi.html)
Tools to analyze & visualize Illumina Infinium methylation arrays.


